index.html

```diff
@@ -4235,14 +4235,10 @@ <h3>Token Binding</h3>
       <p>
 A <a>verifier</a> might need to ensure it is the intended recipient of a
 <a>verifiable presentation</a> and not the target of a
-<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">
-man-in-the-middle attack</a>. Any protocol using the Verifiable Credentials
-Data Model that requires protection against these kinds of attacks needs to
-perform some sort of token binding, such as
-<a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol-18">
-The Token Binding Protocol v1.0</a>, which ties the request for a
-<a>verifiable presentation</a> with the response. Any protocol that does not
-perform token binding is susceptible to man-in-the-middle attacks.
+<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">man-in-the-middle attack</a>.
+Approaches such as token binding [[RFC8471]], which ties the request for a 
```

Comment:
Should this be [[?RFC8471]], as this is a non-normative section and thus this should be an informative reference?